build.zig

```diff
@@ -42,6 +43,12 @@ pub fn build(b: *std.build.Builder) !void {
         b.option(std.log.Level, "log_level", "The Log Level to be used.") orelse .info,
     );
 
+    exe_options.addOption(
+        bool,
+        "pie",
+        pie,
+    );
+
```

Comment:
no reason for this part to be here but otherwise good

Reply:
you're right! PR is updated.